docs/gaia/gaiacli.md

```diff
@@ -608,6 +608,48 @@ gaiacli query gov param tallying
 gaiacli query gov param deposit
 ```
 
+### Distribution
+
+#### Query distribution parameters
+
+To check the current distribution parameters, run:
+
+```bash
+gaiacli query distr params
+```
+
+#### Query outstanding rewards
+
+To check the current outstanding (un-withdrawn) rewards, run:
+
+```bash
+gaiacli query distr outstanding-rewards
```

Comment:
This returns `null` on localnet...perhaps we can return an error or something else...more human-readable?

Reply:
Please see https://github.com/cosmos/cosmos-sdk/pull/3301 for outputs!

Reply:
Probably it returns `null` because you set tokens too low and don't have any inflation (but still `outstanding_rewards` could be zero so we could choose a different serialized zero-value).